event-log-verification-service/src/main/scala/com/ubirch/verification/services/eventlog/DefaultEventLogClient.scala

```diff
@@ -128,7 +129,19 @@ class DefaultEventLogClient @Inject() (finder: Finder)(implicit ec: ExecutionCon
                 }
 
                 blockchainInfo
-                  .map(bi => vertex.addProperties(bi))
+                  .map(bi => {
+                    bi.getOrElse("public_chain", "") match {
```

Comment:
this won't work because

1. iota could come from a tesnet or mainnet
2. vertex is immutable and you are not returning the copy